src/Sdk/HistoryApi.php

```diff
@@ -32,6 +36,89 @@ public function listOperations(
         return $this->dcoreApi->requestWebsocket(History::class, new GetAccountHistory($accountId, $startId, $endId, $limit)) ?: [];
     }
 
+    /**
+     * @inheritdoc
+     */
+    public function listTransferOperationsComposed(
+        ChainObject $accountId,
+        string $startId = '0.0.0',
+        string $endId = '0.0.0',
```

Comment:
Is endId optional? In paging client should sent only startId and limit.

Reply:
Yes it is optional, can be dropped all together if required.

Reply:
Optional can stay.